types/mustache/index.d.ts

```diff
@@ -249,6 +249,101 @@ declare class MustacheWriter {
      * If the template doesn't use higher-order sections, this argument may be omitted.
      */
     renderTokens(tokens: string[], context: MustacheContext, partials?: PartialsOrLookupFn, originalTemplate?: string): string;
+
+    /**
+     * Renders a section block.
+     *
+     * @param token
+     * The token to render.
+     *
+     * @param context
+     * The context to use for rendering the token.
+     *
+     * @param partials
+     * The partials to use for rendering the token.
+     *
+     * @param originalTemplate
+     * An object used to extract the portion of the original template that was contained in a higher-order section.
+     */
+    renderSection(token: string, context: MustacheContext, partials?: PartialsOrLookupFn, originalTemplate?: string): string;
```

Comment:
> **TLDR;** one token is an array of strings.

This makes so much more sense! I was really confused when reading the code, but assumed I understood it wrong because of types for `renderTokens` using `string[]`. I'll change it to `string[][]` here until there's a better solution.

Reply:
Good idea, that's surely better than it has been until now.